dagger-reflect/reflect/src/main/java/dagger/reflect/ReflectiveMembersInjector.java

```diff
@@ -106,4 +116,16 @@ private ReflectiveMembersInjector(
       tryInvoke(instance, methodBinding.getKey(), arguments);
     }
   }
+
+  private static class Bindings {
+    final Map<Field, Binding<?>> fieldBindings;
+    final Map<Method, Binding<?>[]> methodBindings;
```

Comment:
Could've used `Pair`, but neither Android nor Kotlin is on the classpath of this module.
Would it worth using AutoValue here?

Reply:
Don't bother. This is fine.